scripts/triggered_emails.py

```diff
@@ -0,0 +1,67 @@
+import sys
+import logging
+from datetime import datetime, timedelta
+
+from modularodm import Q
+
+from framework.transactions.context import TokuTransaction
+from website.app import init_app
+from website import mails, settings
+from framework.auth import User
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+def main(dry_run=True):
```

Comment:
This needs to be tested -- especially if it will be running on a regular basis. You can look at the tests I wrote for embargoing and retracting registration scripts if you need any pointers.